lib/node-modules-paths.js

```diff
@@ -20,25 +20,37 @@ module.exports = function nodeModulesPaths(start, opts) {
         }
     }
 
-    var prefix = '/';
-    if ((/^([A-Za-z]:)/).test(absoluteStart)) {
-        prefix = '';
-    } else if ((/^\\\\/).test(absoluteStart)) {
-        prefix = '\\\\';
-    }
+    var dirs = [];
+
+    if (!opts || typeof opts.useNodeModules === 'undefined' || opts.useNodeModules) {
```

Comment:
Renamed the option - minimizing the diff would involve doing unnecessary operations at runtime (or breaking the indentation), I'd like to avoid that :/

Reply:
it seems like we could maybe use early returns here, or extract pieces into helper functions?

Reply:
I'm not sure it would drastically improve the code, but if you feel strongly I can do it. What I like with the current formatting is that the order in which the transformations are applied is quite clear - a random user can easily follow it from beginning to end. Early return could make this slightly harder, what do you think?